stdlib/headernt.c

```diff
@@ -93,7 +93,18 @@ static BOOL WINAPI ctrl_handler(DWORD event)
 #define CP CP_THREAD_ACP
 #endif
 
-#define msg_and_length(msg) msg , (sizeof(msg) - 1)
+static void write_console(HANDLE hOut, WCHAR *wstr)
+{
+  DWORD consoleMode, numwritten, len;
+  static char str[MAX_PATH];
+
+  if (GetConsoleMode(hOut, &consoleMode) != 0) { /* The output stream is a Console */
+    WriteConsoleW(hOut, wstr, wcslen(wstr), &numwritten, NULL);
```

Comment:
Out of curiosity: do we actually need the 'W' suffix here?  I thought we were now compiling in a mode where unprefixed functions are aliases for *W ones.

Reply:
I think you are right.  Will fix.